pkg/sql/plan/function/builtin/multi/uuid.go

```diff
@@ -40,3 +40,23 @@ func UUID(inputVecs []*vector.Vector, proc *process.Process) (*vector.Vector, er
 	resultVector := vector.NewWithStrings(types.T_varchar.ToType(), results, nil, proc.Mp())
 	return resultVector, nil
 }
+
+/*
+ UUID function returns the implementation of UUID type
```

Comment:
Do you want this code or not?   I suggest remove this from PR.    Add it when it is really used/tested.

Reply:
OK, I'll delete them, Dr. Tian